### PR TITLE
polish readme for yolov3

### PR DESCRIPTION
**Pull request by tink2123**
- adjust the order of documents
- add data catalog structure

### CODE REVIEW

**Comment by heavengate** [comment]
fix https://github.com/PaddlePaddle/models/issues/2041

File: PaddleCV/yolov3/README.md [hunk 1]
```diff
@@ -45,34 +44,35 @@ Train the model on [MS-COCO dataset](http://cocodataset.org/#download), download
     cd dataset/coco
     ./download.sh
 
+The data catalog structure is as follows:
+
+```
+  data/coco/
```
Comment (heavengate) [inline PaddleCV/yolov3/README.md]: data -> dataset

File: PaddleCV/yolov3/README.md [hunk 2]
```diff
@@ -81,6 +81,28 @@ To train the model, [cocoapi](https://github.com/cocodataset/cocoapi) is needed.
...
+       --pretrain=${path_to_pretrain_model}
+       --data_dir=${path_to_data}
+
+- Set ```export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7``` to specifiy 8 GPU to train.
+- For more help on arguments:
+
+    python train.py --help
+
+
+
```
Comment (heavengate) [inline PaddleCV/yolov3/README.md]: 删除多余空行

File: PaddleCV/yolov3/README.md (hunk 1 above)
Comment (tink2123) [reply PaddleCV/yolov3/README.md]: done

File: PaddleCV/yolov3/README.md (hunk 2 above)
Comment (tink2123) [reply PaddleCV/yolov3/README.md]: done

**Review by heavengate — APPROVED** [review]
LGTM

**Pull request merged**